scripts/shared/clusters.sh

```diff
@@ -16,6 +27,7 @@ eval set -- "${FLAGS_ARGV}"
 
 k8s_version="${FLAGS_k8s_version}"
 olm_version="${FLAGS_olm_version}"
+[[ -n "${kind_k8s_versions[$k8s_version]}" ]] && k8s_version="${kind_k8s_versions[$k8s_version]}"
```

Comment:
```suggestion
[[ -z "${kind_k8s_versions[$k8s_version]}" ]] || k8s_version="${kind_k8s_versions[$k8s_version]}"
```

I think otherwise it breaks (bash is weird sometimes)

Reply:
Do you know under what circumstances?

Reply:
If you’re referring to `set -e`, that doesn’t matter here — the `[[ -n ... ]]` test does produce a non-zero exit code, and then the overall pipeline has a non-zero exit code, but the `&&` causes it to be considered as consumed, and the shell script _doesn’t_ exit as a result.

Reply:
IDK had problems with that before but if it works fine with me :)